src/cml.js

```diff
@@ -143,59 +143,57 @@ class CML {
     return await getDriver(this).runnerToken();
   }
 
-  parseRunnerLog(opts = {}) {
+  async parseRunnerLog(opts = {}) {
     let { data } = opts;
     if (!data) return;
 
+    const date = new Date();
+
     try {
       data = data.toString('utf8');
 
       let log = {
         level: 'info',
-        time: new Date().toISOString(),
+        date: date.toISOString(),
```

Comment:
Renaming `time` to `date` would [break the data format](https://github.com/iterative/terraform-provider-iterative/blob/d7e00bbf2526e75a305588d300d9aadee9a0e976/iterative/utils/runner.go#L12) but it won't not have any short term impact because we don't use that field.

Reply:
@DavidGOrtega, how has this conversation been resolved?